guides/common/modules/proc_performing-post-upgrade-tasks.adoc

```diff
@@ -1,6 +1,9 @@
 [id="performing-post-upgrade-tasks_{context}"]
 = Performing post-upgrade tasks
 
+* Optional: If you are utilizing the oVirt compute resource, it's crucial to migrate your data by executing the `foreman-rake ovirt:drop` task.
+Remember that with the upcoming release of version 3.16, oVirt will be officially phased out, and this migration will be automatically managed during the upgrade process.
```

Comment:
I wonder if we should avoid advertising specific changes for a particular release (because plans can change). May I suggest wording that's a bit more on the vague side, like "In a future version"?

```suggestion
In a future version, {oVirt} will be officially phased out, and this migration will be automatically managed during the upgrade process.
```